docs/gather_data.rst

```diff
@@ -60,6 +60,91 @@ To gather data for the last N days or N hours, use ``--since``:
     $ # see data for the last 12 hours
     $ ./multivac/gather_job_data.py --since 12h
 
+Writing data to JSON file
+-------------------------
+
+To write data to JSON file, use ``--format json`` option. If you add `-t` option,
+script will write data about tests too:
+
+.. code-block:: console
+
+    {
+    "8374189766": {
+    "job_id": 8374189766,
+    "workflow_run_id": 3061077644,
+    "job_name": "out_of_source",
+    "branch": "master",
+    "commit_sha": "416500fed508968d6d890eb3ec3620ef954d1d0a",
+    "conclusion": "success",
+    "queued_at": "2022-09-15T14:02:44Z",
+    "started_at": "2022-09-15T14:02:56Z",
+    "completed_at": "2022-09-15T14:08:45Z",
+    "platform": "amd64",
+    "runner_label": [
+      "ubuntu-20.04-self-hosted"
+    ],
```

Comment:
this part is nested, so it should be indented. Just use any JSON formatter.

```suggestion
      "job_id": 8374189766,
      "workflow_run_id": 3061077644,
      "job_name": "out_of_source",
      "branch": "master",
      "commit_sha": "416500fed508968d6d890eb3ec3620ef954d1d0a",
      "conclusion": "success",
      "queued_at": "2022-09-15T14:02:44Z",
      "started_at": "2022-09-15T14:02:56Z",
      "completed_at": "2022-09-15T14:08:45Z",
      "platform": "amd64",
      "runner_label": [
        "ubuntu-20.04-self-hosted"
      ],
```

Reply:
done